tests/unit/compiler/test_source_map.py

```diff
@@ -31,20 +32,25 @@ def foo(a: uint256) -> int128:
     """
 
 
-def test_jump_map():
+def test_jump_map(optimize):
     source_map = compile_code(TEST_CODE, output_formats=["source_map"])["source_map"]
     pos_map = source_map["pc_pos_map"]
     jump_map = source_map["pc_jump_map"]
 
-    assert len([v for v in jump_map.values() if v == "o"]) == 1
+    expected_jumps = 3 if optimize == OptimizationLevel.NONE else 1
```

Comment:
nit: break into a second line
```suggestion
    expected_jumps = 1
    if optimize == OptimizationLevel.NONE:
        expected_jumps = 3  # some jumps get optimized out when optimizer is on
```